### DISCUSSION TITLE
Albedo and SoilColor in Noah-MP

### DISCUSSION THREAD

**Discussion by manel-bravo** [NCAR/noahmp#129] [Q&A]
Hello,
looking at the code, seem like the Ground Albedo varies between two: a dry and a wet albedo. Those are defined by a table that depends on the soil colour.  And that soil colour seem to be set as a constant in the model

https://github.com/NCAR/noahmp/tree/master/drivers/hrldas/ConfigVarInTransferMod.F90
noahmp%config%domain%SoilColor                   =  4

Looking at the parameters in the NOAHMPTABLE.TBL
 !------------------------------------------------------------------------------
 ! soil color:    1       2       3       4       5       6       7       8     soil color index for soil albedo
 !------------------------------------------------------------------------------
 ALBSAT_VIS =   0.15,   0.11,   0.10,   0.09,   0.08,   0.07,   0.06,   0.05    ! saturated soil albedo at visible band
 ALBSAT_NIR =   0.30,   0.22,   0.20,   0.18,   0.16,   0.14,   0.12,   0.10    ! saturated soil albedo at NIR band
 ALBDRY_VIS =   0.27,   0.22,   0.20,   0.18,   0.16,   0.14,   0.12,   0.10    ! dry soil albedo at visible band
 ALBDRY_NIR =   0.54,   0.44,   0.40,   0.36,   0.32,   0.28,   0.24,   0.20    ! dry soil albedo at NIR band

So the Albedo_VIS varies between 0.09 and 0.18, and the Albedo NIR between 0.18 and 0.36.

Are these values accurate for arid and semi-arid regions? 
Is there any plans to include a soil colour database as input to the model?
Would it not be better to get some initial background albedo from climatology or satellite observations?

I would be thankful to you if you could provide me some insight about this matter.

**Comment by cenlinhe** [comment]
Hi, currently the soil color in Noah-MP has been prescribed to category of 4 uniformly to represent a mean soil color globally. However, this may not work for specific regions. We have not tested it over arid and semi-arid regions, and you may want to optimize it to match with your observations. You could certainly use satellite data to overwrite the background albedo in your case. We have plans to incorporate a global 2-D soil color map inferred from satellite albedo into Noah-MP soon.

**Comment by AlbaraaKhayat** [comment]
Hello, 
We've manually matched the 19 soil types to soil colors and are using this as temporary solution.
This is the table if it might help.
|            | !  1: SAND | !  2: LOAMY SAND | !  3: SANDY LOAM | !  4: SILT LOAM | !  5: SILT | !  6: LOAM | !  7: SANDY CLAY LOAM | !  8: SILTY CLAY LOAM | !  9: CLAY LOAM | ! 10: SANDY CLAY | ! 11: SILTY CLAY | ! 12: CLAY | ! 13: ORGANIC MATERIAL | ! 14: WATER | ! 15: BEDROCK | ! 16: OTHER(land-ice) | ! 17: PLAYA | ! 18: LAVA | ! 19: WHITE SAND |
|------------|------------|------------------|------------------|-----------------|------------|------------|-----------------------|-----------------------|-----------------|------------------|------------------|------------|------------------------|-------------|---------------|-----------------------|-------------|------------|------------------|
| SOIL TYPE: |          1 |                2 |                3 |               4 |          5 |          6 |                     7 |                     8 |               9 |               10 |               11 |         12 |                     13 |          14 |            15 |                    16 |          17 |         18 |               19 |
| SOIL COLOR |          0 |                1 |                2 |               4 |          2 |          7 |                     3 |                     4 |               4 |                1 |                2 |          3 |                      7 |           3 |             7 |                     3 |           0 |          7 |                0 |

I would also like to tune this solution by scaling the ALBSAT_VIS, ALBDRY_VIS values.

**Reply by cenlinhe** [reply]
thank you. this table is helpful.

**Reply by cenlinhe** [reply]
by the way, how do you assign different soil colors to soil types? is there any reference you used? If so, could you provide it here? We would like to document it. thanks!

**Reply by AlbaraaKhayat** [reply]
I have no reference, but soil types are sorted by perceived darkness on a scale of 0 to 7. 
This is an arbitrary hack meant only to be used until an alternative (like referencing soil types to albedo data) is available.
We added the column to soilparams table and use it for ALBSOD in GROUNDALB (v4.5).
